### ISSUE TITLE
Proposal:  ≢-sym

### ISSUE THREAD

**Issue by mechvel** [agda/agda-stdlib#670]
I have an impression that  Relation.Binary.PropositionalEquality needs
```
≢-sym :  Symmetric _≢_
```

**Comment by mechvel** [comment]
Also  IsEquivalence 
needs to include this implementation part:
```
_≉_ :  Rel A _
_≉_ x =   ¬_ ∘ _≈_ x

≉-sym :  Symmetric _≉_
≉-sym {x} {y} x≉y =  x≉y ∘ ≈sym

≉-cong :  _≉_ Respects2 _≈_                             -- ... x ≉ y → x' ≉ y'  
≉-cong {x} {x'} {y} {y'} x=x' y=y' x≉y x'=y' =  x≉y x=y
                                           where
                                           x=y =  trans (trans x=x' x'=y') (sym y=y')

≉congˡ :  {y : Carrier} → (_≉ y) Respects _≈_
≉congˡ = ...

≉congʳ :  {x : Carrier} → (x ≉_) Respects _≈_
≉congʳ  = ...
```
This  a) helps usage of ≉, b) is backwards compatible, c) does not introduce new data fields.

**Comment by gallais** [comment]
That sounds like a plan. Could we have more generic statements in
`Relation.Nullary.Negation` and then specific instances in
`Relation.Nullary.Negation.(Setoid/Propositional)`?

**Comment by MatthewDaggitt** [comment]
I agree these are useful.

> Could we have more generic statements in
Relation.Nullary.Negation and then specific instances in
Relation.Nullary.Negation.(Setoid/Propositional)?

Hmm it seems like these should go in `Relation.Binary.Properties.Setoid`? (doesn't exist yet!)

**Comment by MatthewDaggitt** [comment]
Closed with e25f109
